pkg/bucketclaim/bucketclaim.go

```diff
@@ -153,7 +166,10 @@ func (b *bucketClaimListener) provisionBucketClaimOperation(ctx context.Context,
 		}
 
 		bucketClass, err := b.bucketClasses().Get(ctx, bucketClassName, metav1.GetOptions{})
-		if err != nil {
+		if kubeerrors.IsNotFound(err) {
+			b.recordEvent(inputBucketClaim, v1.EventTypeWarning, "InvalidBucketClass", "BucketClass provided in the BucketClaim does not exist")
```

Comment:
One of the things Rook does is codify the event types  in it's API so that applications using Rook programmatically can see the definitions. Should we do this with COSI as well? This would be "InvalidBucketClass" here.

At any rate, it seems best to use a const definition for InvalidBucketClass.